pkg/ebpf/netlink/conntracker_test.go

```diff
@@ -152,19 +152,24 @@ func TestGetUpdatesGen(t *testing.T) {
 
 	rt.register(c)
 	var last uint8
+
+	// there will be two entries in rt.state for the entry we just registered.
+	// set them both to be 5 generations older than they are
+	require.Len(t, rt.state, 2)
 	for _, v := range rt.state {
```

Comment:
If there is more than one entry, will we get bit by the random iteration order?

Reply:
Iteration order will be random, but we want to set the generation to current-5 for all entries.